go-controller/pkg/ovn/egressip_test.go

```diff
@@ -271,12 +271,6 @@ var _ = ginkgo.Describe("OVN master EgressIP Operations", func() {
 				gomega.Eventually(isEgressAssignableNode(node1.Name)).Should(gomega.BeTrue())
 				gomega.Eventually(isEgressAssignableNode(node2.Name)).Should(gomega.BeFalse())
 
-				fakeOvn.fakeExec.AddFakeCmdsNoOutputNoError(
```

Comment:
Could you please not just delete these commands, but also change the tests to verify that the NAT objects are created/removed?

Reply:
Hmm... libovsdbtest.TestData already ensures that, no? Are you are proposing we add more tests to verify that?

Reply:
I don't quite follow. `libovsdbtest.TestData` only asserts that whatever you expect to have in the DB at the end of the test, also exists in the DB. However you have not added any lines asserting that NAT objects exists / do not exists at the end of each test. An example would be:

1. Initialize an empty NB DB at the start of the test
2. Create an egress IP
3. Assert that the NAT objects required are there at the end of the test. 

This is just an example that doesn't necessarily apply to this particular test case. But you will need to read the ovn-nbctl commands that were existing, see what they expected to happen and then write an assertion based on that of what you expect the NB DB to have at the end of the test.

Reply:
Ack! I will work on that next.

Reply:
Done! PTAL